wagtailcaptcha/models.py

```diff
@@ -18,14 +18,10 @@ def __init__(self, *args, **kwargs):
         self.form_builder = WagtailCaptchaFormBuilder
 
     def process_form_submission(self, form):
-        super(AbstractEmailForm, self).process_form_submission(form)
-
-        if self.to_address:
-            content = ''
-            for x in form.fields.items():
-                if not isinstance(x[1], ReCaptchaField):  # exclude ReCaptchaField from notification
-                    content += '\n' + x[1].label + ': ' + text_type(form.data.get(x[0]))
-            send_mail(self.subject, content, [self.to_address], self.from_address,)
+        if WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME in form.fields.keys():
+            form.fields.pop(WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME)
+
+        return super(WagtailCaptchaEmailForm, self).process_form_submission(form)
```

Comment:
I was wrong about this.

```    
def process_form_submission(self, form):
        FormSubmission.objects.create(
            form_data=json.dumps(form.cleaned_data, cls=DjangoJSONEncoder),
            page=self,
        )
``` 

Parent's class uses form.cleaned_data, so removing the field from form.fields will not remove the `wagtailcaptcha` from database. It will only remove from the e-mail sent. Based on this, we don't need to propagate it to `WagtailCaptchaForm`.


I was wrong about this.

```    
def process_form_submission(self, form):
        FormSubmission.objects.create(
            form_data=json.dumps(form.cleaned_data, cls=DjangoJSONEncoder),
            page=self,
        )
``` 

Parent's class uses form.cleaned_data, so removing the field from form.fields will not remove the `wagtailcaptcha` from database. It will only remove from the e-mail sent. Based on this, we don't need to propagate it to `WagtailCaptchaForm`.